pgsql/pointcloud.sql.in

```diff
@@ -309,6 +309,10 @@ CREATE OR REPLACE FUNCTION PC_IsSorted(p pcpatch, attr text[], strict boolean de
 	RETURNS boolean AS 'MODULE_PATHNAME', 'pcpatch_is_sorted'
 	LANGUAGE 'c' IMMUTABLE STRICT;
 
+CREATE OR REPLACE FUNCTION PC_SetSchema(p pcpatch, pcid int4)
+	RETURNS pcpatch AS 'MODULE_PATHNAME', 'pcpatch_setschema'
+	LANGUAGE 'c' IMMUTABLE STRICT;
+
```

Comment:
I would rather have the reprojection be performed by a `PC_Transform` function and have the `PC_SetPCId/PC_SetSchema` function not do any reprojection (only attribute raw type casting including scale/offset modification, discarding attributes, adding attributes with default values, and attribute reordering)